index.js

```diff
@@ -4,20 +4,27 @@ const md5 = require("md5-file");
 const camelCase = require("camelcase");
 const fs = require("fs-extra");
 const handlebars = require("handlebars");
+const isString = x => typeof x === "string";
+const isFunction = x => typeof x === "function";
 
 function collectAssets(config, callback) {
-  const { assetsPath, replacePath, buildUrl } = config;
+  const {
+    assetsPath,
+    replacePath,
+    buildUrl
+  } = config;
   let assets = [];
   try {
     dive(
       assetsPath,
-      (err, file) => {
+      (err, source) => {
         if (err) throw new Error(err);
 
-        const fileName = replacePath(file);
-        const hash = md5.sync(file);
-        const url = buildUrl(fileName, hash);
-        const elmName = createElmName(fileName);
+        const dest = replacePath(source);
+        const hash = md5.sync(source);
+        const url = buildUrl(dest, hash);
```

Comment:
💭 I might rename `url` to something like `urlWithHash` for a reminder of what the difference is later